src/python/deepgnn/graph_engine/snark/distributed.py

```diff
@@ -1,18 +1,37 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-"""Snark districuted client implementation."""
-from typing import List, Dict
+"""
```

Comment:
why docstring and classes are copy pasted from [synchronized.py](https://github.com/microsoft/DeepGNN/blob/main/src/python/deepgnn/graph_engine/backends/snark/synchronized.py)?

Reply:
Fixed